frontend/lib/Views/compare_endpoints_view.dart

```diff
@@ -48,36 +50,52 @@ class _CompareChartsViewState extends State<CompareChartsView> {
         child: RefreshIndicator(
           onRefresh: () => _pullDownRefresh(),
           child: Scaffold(
-            appBar: buildAppBar(compareEndpointsViewAppBar),
-            body: FutureBuilder<List<EndpointSummary>>(
-              future: widget.endpointGateway.getEndpointSummary(),
-              builder: (context, snapshot) {
-                if (snapshot.connectionState == ConnectionState.none ||
-                    snapshot.data == null) {
-                  return loadingInCenter();
-                } else {
-                  return SingleChildScrollView(
-                    child: Column(
-                      children: [
-                        buildDropDownSelection(context, snapshot),
-                        const SizedBox(height: 10),
-                        Consumer<CompareEndpointsModel>(
-                          builder: (context, endpointModel, child) => Wrap(
-                            children: _createChips(endpointModel),
+            appBar: buildFancyAppBar(compareEndpointsViewAppBar),
+            body: Container(
+              height: MediaQuery.of(context).size.height,
+              decoration: const BoxDecoration(
+                gradient: LinearGradient(
+                  begin: Alignment.topRight,
+                  end: Alignment.bottomLeft,
+                  colors: [
+                    Color.fromRGBO(21, 184, 194, 1),
+                    Color.fromRGBO(14, 14, 82, 0.9)
+                  ],
+                ),
+              ),
```

Comment:
Imo całe  `const BoxDecoration` wrzuciłbym do jakiegoś pliku `widgets_decorations`, bo to tło zapewne będzie się powtarzało w wielu miejscach